src/Sulu/Bundle/SecurityBundle/Behat/SecurityContext.php

```diff
@@ -136,6 +160,31 @@ public function theNotEnabledUserExistsWithPassword($username, $password)
     public function iAmLoggedInAsAnAdministrator()
     {
         $this->theUserExistsWithPassword('admin', 'admin');
+
+        $this->visitPath('/admin');
+        $page = $this->getSession()->getPage();
+        $this->waitForSelector('#username');
+        $this->fillSelector('#username', 'admin');
+        $this->fillSelector('#password', 'admin');
+        $loginButton = $page->findById('login-button');
+
+        if (!$loginButton) {
+            throw new \InvalidArgumentException(
+                'Could not find submit button on login page'
+            );
+        }
+
+        $loginButton->click();
+        $this->getSession()->wait(5000, "document.querySelector('.navigation')");
```

Comment:
In my opinion not the prettiest way because we don't have a clear defined step (I am logged in as an administrator with locale 'de'). Also i need to check if the user exists.